code/modules/projectiles/projectile/special.dm

```diff
@@ -141,10 +141,14 @@
 	nodamage = 1
 	flag = "energy"
 
-/obj/item/projectile/beam/mindflayer
+/obj/item/projectile/energy/mindflayer
```

Comment:
This seems like an undocumented change

Reply:
oops, i forgot to mention this, thanks for pointing it out